src/components/Counter.js

```diff
@@ -6,11 +6,29 @@ class Counter extends Component {
     incrementIfOdd = () => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+        // takes the current count divides it by 2 and increases the count if the remainder is 0
+        if( this.props.count % 2 !== 0 ){
```

Comment:
I'm very happy that you pushed for the stretched, and you did an awesome job explaining it to Robert